mirrord/cli/src/config.rs

```diff
@@ -190,6 +190,10 @@ pub(super) enum Commands {
 
     /// Execute a command related to mirrord CI.
     Ci(Box<CiArgs>),
+    
+    /// Launch the config wizard
```

Comment:
+ a brief description of what the wizard is? Something like:

```suggestion
    /// Launch the config wizard.
    ///
    /// The mirrord config wizard is a web app that guides you into building a mirrord.json config file, providing you with a premium, curated experience ...
```